sync.go

```diff
@@ -39,38 +39,48 @@ type syncData struct {
 type activeSyncData struct {
 	syncStage int32
 
+	cfiltersFetchProgress    CFiltersFetchProgressReport
 	headersFetchProgress     HeadersFetchProgressReport
 	addressDiscoveryProgress AddressDiscoveryProgressReport
 	headersRescanProgress    HeadersRescanProgressReport
 
-	beginFetchTimeStamp   int64
-	startHeaderHeight     int32
-	headersFetchTimeSpent int64
-
-	addressDiscoveryStartTime int64
-	totalDiscoveryTimeSpent   int64
-
 	addressDiscoveryCompletedOrCanceled chan bool
 
 	rescanStartTime int64
 
-	totalInactiveSeconds     int64
-	totalFetchedHeadersCount int32
+	totalInactiveSeconds int64
 }
 
 const (
 	InvalidSyncStage          = -1
-	HeadersFetchSyncStage     = 0
-	AddressDiscoverySyncStage = 1
-	HeadersRescanSyncStage    = 2
+	CFiltersFetchSyncStage    = 0
+	HeadersFetchSyncStage     = 1
+	AddressDiscoverySyncStage = 2
+	HeadersRescanSyncStage    = 3
```

Comment:
you can use the iota enum type for successive integer increments

Reply:
iota enum would not be visible when compiled with gomobile which is why it's written this way.